src/_pluggy/hookspec.py

```diff
@@ -0,0 +1,17 @@
+import pluggy
```

Comment:
You are missing a standard header and the `__future__` import including the unicode ones